app/services/data_migrations/remove_ske_feature_flag.rb

```diff
@@ -0,0 +1,10 @@
+module DataMigrations
+  class RemoveSkeFeatureFlag
+    TIMESTAMP = 20230502113523
+    MANUAL_RUN = false
+
+    def change
+      Feature.where("name = 'ske' OR name = 'provider_ske'").delete_all
```

Comment:
```suggestion
      Feature.where(name: %w(ske provider_ske)).delete_all
```

Reply:
just cos i know you want to be maximally cool

Reply:
:)